Marlin/Marlin_main.cpp

```diff
@@ -2424,12 +2424,6 @@ static void clean_up_after_endstop_or_probe_move() {
         do_blocking_move_to_z(delta_clip_start_height);
     #endif
 
-    #if HAS_SOFTWARE_ENDSTOPS
```

Comment:
Testing was done for printable = false that the nozzle stays within the printable radius; only the probe can move outside it (no testing on the probe with printable = false); so this is obsolete.